config/job.yaml.tim

```diff
@@ -39,9 +44,25 @@ spec:
         volumeMounts:
         - mountPath: /opt/screwdriver
           name: screwdriver
+        - mountPath: /var/run/sd
+          name: logs
         env:
         - name: SD_TOKEN
           value: "{{token}}"
+      - name: logservice
+        image: golang:1.7
```

Comment:
If we want to build the program on launch. I assume we don't want to do that.

We could easily use Alpine like job-tools https://github.com/screwdriver-cd/job-tools/blob/master/Dockerfile#L1

We probably need to do that glibc hack to make it work